src/site/content/en/measure/index.njk

```diff
@@ -22,6 +22,9 @@ to remove them as part of the v1 migration. #}
         </span>
       </p>
       <web-signin-button></web-signin-button>
+      <p class="w-page-header__copy w-unsupported-notice">
+        Measure isn't available on your browser. Try a supported browser.
```

Comment:
I can already feel the grumpy tweets 😁

Maybe tweak the language to say. "Sorry, the measure tool does not work with legacy browsers. Please try again using the latest Firefox, Safari, or Chrome."

Reply:
Sounds good.